components/slider/index.css

```diff
@@ -455,17 +495,239 @@ governing permissions and limitations under the License.
 /* backwards compatibility: these are not required, but they make the handle go to the edegs and align right */
 .spectrum-Slider-handleContainer,
 .spectrum-Slider-trackContainer {
-  width: calc(100% + var(--spectrum-slider-handle-width));
+  inline-size: calc(100% + var(--spectrum-slider-handle-size));
 
   position: absolute;
-  top: calc(var(--spectrum-slider-track-height) / 2 - 1px);
+  inset-block-start: calc(var(--mod-slider-control-height, var(--spectrum-slider-control-height)) / 2 - 1px);
```

Comment:
Lines 414 and 501 are causing the tick variant to have extra space from label to controls. When I comment out lines 414 and 501, they move to where they are supposed to be, but the tick positions will need to be adjusted:

![Screenshot 2023-02-03 at 4 36 28 PM](https://user-images.githubusercontent.com/25614178/216724196-fc3e343c-0ba3-4918-ab15-07891f97ccdb.png)

And likewise the track and handle position will need to be adjusted as well:
![Screenshot 2023-02-03 at 4 37 15 PM](https://user-images.githubusercontent.com/25614178/216724329-3a989907-da3b-4725-b2a2-31723cd4d0db.png)